### PR TITLE
chore: Make some icons clearer and add some media queries

### LINKED ISSUE #944: feat: add user page
### Current State

Now we have the powered-by page, which already corresponds to some icons. But they don't look uniform enough.

powered-by: https://github.com/apache/apisix/blob/master/powered-by.md

### Desired State

Maybe we can refer to the user cases design of skywalking website: https://skywalking.apache.org/users/

link: https://github.com/apache/skywalking-website/blob/master/data/users.yml

### Would you like to fix this issue?

_No response_

### PR DESCRIPTION
**Pull request by hanzhenfang**
Fixes: #944

Changes: Make some icons clearer and add some Media queries.

hey, I have replaced nine pictures like these.
The old:
imageimageimageimageimageimage

The new:
imageimageimageimageimageimage

And so on.
I also add some Media queries to make the page look better on a small screen like this.
The old:

image

The new
image

### CODE REVIEW

**Comment by guoqqqi** [comment]
image
Please note the spacing between the title

File: website/src/pages/showcase/styles.scss [hunk 1]
```diff
@@ -35,7 +35,16 @@
...
     }
   }
+
+@media (max-width: 820px) {
+  .content {
+    display: grid;
+    grid-template-columns: 1fr 1fr;
+  }
 }
+}
```
Comment (guoqqqi) [inline website/src/pages/showcase/styles.scss]: We usually add a blank line at the end of the file

File: website/static/data/showcases.json [hunk 2]
```diff
@@ -401,4 +401,4 @@
     "infoLink": "https://www.xinpianchang.com",
     "pinned": true
   }
-]
+]
```
Comment (guoqqqi) [inline website/static/data/showcases.json]: ditto

File: website/src/pages/showcase/index.js [hunk 3]
```diff
@@ -43,9 +43,9 @@ const Content = (props) => {
   const showcases = siteConfig.customFields.showcases.map((user) => (
     <div className="col col--2 item" key={user.infoLink}>
       <UserCard
-          image={'https://cdn.jsdelivr.net/gh/apache/apisix-website@master/website/static/img/' + user.image}
-          caption={user.caption}
-          infoLink={user.infoLink}
+        image={'../../../static/img/' + user.image}
```
Comment (guoqqqi) [inline website/src/pages/showcase/index.js]: Here you should continue to use the CDN to speed up the link

File: website/src/pages/showcase/styles.scss (hunk 1 above)
Comment (hanzhenfang) [reply website/src/pages/showcase/styles.scss]: > We usually add a blank line at the end of the file

ok I will improve them

File: website/src/pages/showcase/index.js (hunk 3 above)
Comment (hanzhenfang) [reply website/src/pages/showcase/index.js]: @guoqqqi Hello,I have improved my documents.PTAL.😄

**Comment by yzeng25** [comment]
All checks have passed now ( test and deploy was failed but I re-ran it just now). Pleas take a look @guoqqqi @bzp2010

File: yarn.lock [hunk 4]
```diff
@@ -0,0 +1,4 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
+# yarn lockfile v1
+
+
```
Comment (guoqqqi) [inline yarn.lock]: This file does not need to be uploaded, please remove it

Comment (hanzhenfang) [reply yarn.lock]: > This file does not need to be uploaded, please remove it

Ok

**Review by guoqqqi — DISMISSED** [review]
LGTM

**Review by yzeng25 — DISMISSED** [review]

**Comment by yzeng25** [comment]
LGTM2

File: website/static/data/showcases.json [hunk 5]
```diff
@@ -43,7 +43,7 @@
   },
   {
     "caption": "嘀嗒出行",
-    "image": "showcase/didachuxing.jpg",
+    "image": "showcase/didachuxing.webp",
```
Comment (juzhiyuan) [inline website/static/data/showcases.json]: Hi, you could visit https://caniuse.com/?search=webp to check if a property or feature is supported in different browsers.

image


For webp, it seems that Safari doesn't support it well?

image

Comment (hanzhenfang) [reply website/static/data/showcases.json]: @juzhiyuan 
what about IE?
image

**Comment by hanzhenfang** [comment]
@guoqqqi @juzhiyuan 
I have replaced two WEBP images in JPEG format.
image
image
But I don't know whether I should take about the compatibility of IE.🤔

File: website/src/pages/showcase/styles.scss [hunk 6]
```diff
@@ -35,7 +35,16 @@
...
       padding: 50px 50px;
     }
   }
+
+@media (max-width: 820px) {
+  .content {
+    display: grid;
+    grid-template-columns: 1fr 1fr;
+  }
+}
```
Comment (guoqqqi) [inline website/src/pages/showcase/styles.scss]: format code

File: website/static/data/showcases.json (hunk 5 above)
Comment (juzhiyuan) [reply website/static/data/showcases.json]: Microsoft drops IE, and I support not considering it.

Comment (bzp2010) [reply website/static/data/showcases.json]: Truth be told, I prefer to use svg vector graphics over webp, if icons available. Hopefully we can make it happen in the future.

Comment (hanzhenfang) [reply website/static/data/showcases.json]: > Truth be told, I prefer to use SVG vector graphics over webp, if icons available.

I'm sorry about that.I have tried my best to find these icons of SVG, but it's too hard find them.And some of their website ICONS are not particularly clear.

Comment (bzp2010) [reply website/static/data/showcases.json]: Yes, so I modified it and added the sentence `Hopefully we can make it happen in the future.`

Comment (hanzhenfang) [reply website/static/data/showcases.json]: > Yes, so I modified it and added the sentence `Hopefully we can make it happen in the future.`

Yes, I hope so! If I find them， I will pull the request immediately.😄

**Review by guoqqqi — DISMISSED** [review]
SGTM!

File: website/static/data/showcases.json (hunk 2 above)
Comment (juzhiyuan) [inline website/static/data/showcases.json]:
```suggestion
]

```

**Review by juzhiyuan — DISMISSED** [review]

**Review by juzhiyuan — DISMISSED** [review]

**Review by guoqqqi — DISMISSED** [review]

**Review by yzeng25 — DISMISSED** [review]

**Review by juzhiyuan — APPROVED** [review]

**Review by yzeng25 — APPROVED** [review]

**Comment by lijing-21** [comment]
Hi @hanzhenfang  , thank you for your contribution!

Here is the Contributor T-shirt form[1], if you're interested, kindly take a look :)

[1] https://github.com/apache/apisix/blob/master/CONTRIBUTING.md#contributor-t-shirt

**Comment by hanzhenfang** [comment]
@lijing-21 I'm so happy that I don't know what to say to show my thanks！ I just finished filling out the form.
The last thing to say is I wish our community to become more robust, thank you very much！💗

**Pull request merged**